src/FSharp.ProjectTemplate/paket.template

```diff
@@ -0,0 +1,6 @@
+type project
+authors 
+    ##Author##
+description
+    ##Description##
+
```

Comment:
but wait. I can load the Id from the assembly without custom attributes. give me a minute